content/applications/hr/fleet/service.rst

```diff
@@ -6,72 +6,282 @@ To properly maintain a fleet of vehicles, regular maintenance as well as periodi
 needed. Scheduling repairs and managing services for an entire fleet is necessary to ensure all
 vehicles are in good working order when they are needed.
 
+.. _service/service-form:
+
 Create a service record
 =======================
 
-To log a service, go to the main services dashboard by navigating to :menuselection:`Fleet app -->
-Fleet --> Services`. Click :guilabel:`Create` and a service form appears. Fill in the information on
-the form and click :guilabel:`Save`. The only two fields that are required to be populated are
+To log a service for a veicle, go to the main services dashboard by navigating to
+:menuselection:`Fleet app --> Fleet --> Services`. Click the :guilabel:`New` button in the top-left
+corner, and a new service form loads.
+
+Fill out the information on the form. The only two fields that are required to be populated are
 :guilabel:`Service Type` and :guilabel:`Vehicle`.
 
-The fields are:
+The service form automatically saves as data is entered. However, the form can be saved manually at
+any time by clicking the :guilabel:`Save manually` option, represented by a :guilabel:`cloud upload`
+icon.
+
+The fields on the form are:
 
 - :guilabel:`Description`: enter in a short description of the service.
-- :guilabel:`Service Type`: select the type of service performed from the drop-down menu, or type in
-  a new type of service and click either :guilabel:`Create` or :guilabel:`Create & Edit...`. This
-  list is not pre-configured in Odoo. When creating a fleet and logging a service, the types of
-  service need to be created.
-- :guilabel:`Date`: using the calendar module, select the date the service was provided or is
-  scheduled to be performed. Navigate to the desired month using the left and right arrow icons,
+- :guilabel:`Service Type`: select the type of service performed using the drop-down menu, or enter
+  a new type of service and click either :guilabel:`Create "service type"` or :guilabel:`Create and
+  edit...` to :ref:`add the service type and configure it <service/new-type>`.
```

Comment:
Since you go into such detail about how  to create a service type later, what do you think about simply linking to it? (: Keeps things short and, and you already did the work anyway (;
```suggestion
- :guilabel:`Service Type`: choose from the drop-down menu, or :ref:`add the service type and configure it <service/new-type>`.
```

Reply:
The only reason I do want to keep it is they can just add it without configuring it (that part is not 100% necessary) and then if they do want to configure it, they can jump to those instructions.